itou/archive/management/commands/anonymize_professionals.py

```diff
@@ -45,6 +55,82 @@ def reset_notified_professionals_with_recent_activity(self):
             reset_nb = users_to_reset_qs.count()
         self.logger.info("Reset notified professionals with recent activity: %s", reset_nb)
 
+    @transaction.atomic
+    def anonymize_professionals_after_grace_period(self):
+        now = timezone.now()
+        grace_period_since = now - GRACE_PERIOD
+        self.logger.info("Archiving professionals after grace period, notified before: %s", grace_period_since)
+
+        users_to_anonymize = self.get_user_ids(grace_period_since)
```

Comment:
```suggestion
        users_to_anonymize = self.get_users(grace_period_since)
```
(et renommer la fonction également)